gulp-tasks/documentation.js

```diff
@@ -29,6 +29,8 @@ navigation_weight: 0
 
 `;
 
+const AUTO_GENERATED = "<!-- DO NOT EDIT. This page is autogenerated. -->";
```

Comment:
Lint is throwing an error on this because it's double quote, change to single quote to work.

Test locally by running `gulp lint`.